airflow/ui/src/pages/Variables/Variables.tsx

```diff
@@ -33,27 +33,30 @@ import {
 } from "src/constants/searchParams";
 
 import AddVariableModal from "./AddVariableModal";
+import { VariableActions } from "./VariableAction";
 
 const columns: Array<ColumnDef<VariableResponse>> = [
   {
     accessorKey: "key",
     header: "Key",
-    meta: {
-      skeletonWidth: 25,
-    },
   },
   {
     accessorKey: "value",
     header: "Value",
-    meta: {
-      skeletonWidth: 25,
-    },
   },
   {
     accessorKey: "description",
     header: "Description",
+  },
+  {
+    accessorKey: "actions",
+    cell: ({ row: { original } }) => (
+      <VariableActions variableKey={original.key} />
+    ),
+    enableSorting: false,
+    header: "Actions",
```

Comment:
This should be consistent with clear and trigger actions. Either remove the header everywhere, or put `Actions` everywhere.

I have no strong preferences. Maybe `Actions` everywhere is better.

Reply:
I added the actions header, since everything was wrapped under a button, but if we chose it to stay outside so it will be already visible to user, then maybe we can leave it empty.

Reply:
👍